src/utils/database.js

```diff
@@ -57,6 +57,30 @@ function getConnectionOptions() {
 	};
 }
 
+/**
+ * Splitting the inputs into 3 letters for search indexing.
+ * + first letter
+ * + first two letters combination
+ * to increase the quality
+ *
+ * @param {*} string
+ * @returns [Array]
+ */
+const splitForSearchIndexes = (...searchTexts) => {
```

Comment:
please add tests

Reply:
also: I don't think this is the right place for this function. I would suggest moving it either into its own module or into the helpers directory (src/helper)

Reply:
whats the diffrent between helper and utils? own module means own files in utils, like search.js?